### PR TITLE
Backuppc styling fix

### PR DESCRIPTION
**Pull request by VoyTechnology**
Resolves #19

### CODE REVIEW

File: docs/services/backuppc.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
-Backuppc is currently installed on albus and can be accessed from albus.redbrick.dcu.ie/backuppc login details are in the pw safe
+Backuppc is currently installed on albus and can be accessed from
+[albus.redbrick.dcu.ie/backuppc](http://albus.redbrick.dcu.ie/backuppc).
```
Comment (butlerx) [inline docs/services/backuppc.md]: https please

File: docs/services/backuppc.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
 
 Before a host will be backed up a few things need to be done
 
-  - The machine tobe backed up needs to have its sshd config changed to the following `PermitRootLogin forced-commands-only`
-  - Then the ssh key must been added to the the authorizedkey file in roots .ssh dir. The ssh key can be found in roots .ssh directory on albus. The first part of the file must contain the following 
+- The machine to be backed up needs to have its `sshd` config changed to the
+  following `PermitRootLogin forced-commands-only`
+- Then the ssh key must been added to the the authorizedkey file in roots
+  `.ssh` dir. The ssh key can be found in root's `.ssh` directory on albus.
+  The first part of the file must contain the following
```
Comment (butlerx) [inline docs/services/backuppc.md:11]: can uou leave empty lines above and below the code block for readthedocs

**Review by butlerx — CHANGES_REQUESTED** [review]

File: docs/services/backuppc.md (hunk 1 above)
Comment (VoyTechnology) [reply docs/services/backuppc.md]: Didn't ~~know~~ check is it available

File: docs/services/backuppc.md (hunk 2 above)
Comment (VoyTechnology) [reply docs/services/backuppc.md:11]: Done.

**Review by butlerx — APPROVED** [review]

**Pull request merged**